src/learner/tests/test_learner.py

```diff
@@ -6,6 +6,8 @@
 from learner.learner import UserDocumentsAccumulator, UserData, UserDoc, _similarity_by_row, _FixedSizeHeap
 
 # test implementation of L2 norm without re-using numpy code
+
```

Comment:
autopep8 tool is a little overzealous...Fixed (comment should be inside function in python or it's a considered to be a module level comment I think